sources_test.go

```diff
@@ -155,6 +155,40 @@ func TestListFilesGlob(t *testing.T) {
 	}
 }
 
+func TestFindFilesMultipleFiles(t *testing.T) {
+	t.Parallel()
+	dir := "testdata/multiple_files"
+	want := fmt.Sprintf("%s/1.txt\n%s/2.txt\n%s/3.tar.zip\n", dir, dir, dir)
+	got, err := ListFiles(dir).String()
```

Comment:
We probably want to call `FindFiles` rather than `ListFiles` here, don't we? Or another way to think it about it, if there are no subdirectories, then the result of `FindFiles` should be the same as that of `ListFiles`. But I prefer having an explicit expectation, as you do here.